packages/proxy/src/providers/anthropic.ts

```diff
@@ -157,13 +157,96 @@ function makeAnthropicImageBlock(base64Image: string): AnthropicImageBlock {
   });
 }
 
-export function openAIContentToAnthropicContent(
+const maxImageBytes = 5 * 1024 * 1024;
+const allowedImageTypes = [
+  "image/jpeg",
+  "image/png",
+  "image/gif",
+  "image/webp",
+];
+
+function arrayBufferToBase64(buf: ArrayBuffer): string {
```

Comment:
let's try to find something (either through node compat, or `btoa`) that lets us use an existing implementation

Reply:
`btoa` did the trick